media-proxy/src/session-rdma-tx.cc

```diff
@@ -49,7 +49,6 @@ TxRdmaSession::TxRdmaSession(libfabric_ctx *dev_handle, const mcm_conn_param &re
     ep_cfg.rdma_ctx = dev_handle;
     memcpy(&ep_cfg.remote_addr, &request.remote_addr, sizeof(request.remote_addr));
     memcpy(&ep_cfg.local_addr, &request.local_addr, sizeof(request.local_addr));
-    ep_cfg.dir = direction::TX;
```

Comment:
are you sure that this does not break the current implementation?

Reply:
fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/commit/737483c86a4bf88fdd86b9e223e58acf7f3b7d6d